demo/basic/generated/kotlin/BridgeTypes.kt

```diff
@@ -143,6 +145,16 @@ class OverriddenFullSizeMembersUnionTypeTypeAdapter : JsonSerializer<OverriddenF
         } catch (e: Exception) {
             // Ignore the exception and try the next type
         }
+        try {
+            return OverriddenFullSizeMembersUnionTypeType.StringArrayValue(context.deserialize(json, Array<String>::class.java))
+        } catch (e: Exception) {
+            // Ignore the exception and try the next type
+        }
+        try {
+            return OverriddenFullSizeMembersUnionTypeType.StringForStringDictionaryValue(context.deserialize(json, Map<String, String>::class.java))
```

Comment:
There should be more types, but I think it's ok right now, we can add more when we need. Thanks.